typescript/hardhat/src/TestAbacusDeploy.ts

```diff
@@ -85,36 +112,43 @@ export class TestAbacusDeploy extends TestDeploy<
 
     const inboxFactory = new TestInbox__factory(signer);
     const inboxes: Record<types.Domain, TestInbox> = {};
-    // this.remotes reads this.instances which has not yet been set.
-    const remotes = Object.keys(this.config.signer).map((d) => parseInt(d));
-    const deploys = remotes.map(async (remote) => {
+    const inboxDeploys = remotes.map(async (remote) => {
+      const inboxMultisigValidatorManager =
+        inboxMultisigValidatorManagers[remote];
       const inbox = await inboxFactory.deploy(domain);
       await inbox.initialize(
         remote,
-        validatorManager.address,
+        inboxMultisigValidatorManager.address,
         ethers.constants.HashZero,
         0
       );
       await xAppConnectionManager.enrollInbox(remote, inbox.address);
       inboxes[remote] = inbox;
     });
-    await Promise.all(deploys);
+    await Promise.all(inboxDeploys);
     return {
       outbox,
       xAppConnectionManager,
       interchainGasPaymaster,
-      validatorManager,
       inboxes,
       upgradeBeaconController,
+      outboxMultisigValidatorManager,
+      inboxMultisigValidatorManagers,
     };
   }
 
   async transferOwnership(domain: types.Domain, address: types.Address) {
     await this.outbox(domain).transferOwnership(address);
     await this.upgradeBeaconController(domain).transferOwnership(address);
     await this.xAppConnectionManager(domain).transferOwnership(address);
-    await this.validatorManager(domain).transferOwnership(address);
+    await this.outboxMultisigValidatorManager(domain).transferOwnership(
+      address
+    );
     for (const remote of this.remotes(domain)) {
+      await this.inboxMultisigValidatorManager(
+        domain,
+        remote
+      ).transferOwnership(address);
       await this.inbox(domain, remote).transferOwnership(address);
```

Comment:
are there cases where `inbox` and `inboxMultisigValidatorManager` should have separate `owners`?

Reply:
In a "real" environment I imagine the owner of both will be a governance/controller deployment, and for a test environment like this is creating I'd expect the tests to have the same owner for ease of testing